apps/portals/elportal/src/config/synapseConfigs/handleUpsetPlotClick.tsx

```diff
@@ -1,19 +1,29 @@
-import { Query, FacetColumnRequest } from '@sage-bionetworks/synapse-types'
-import { cohortBuilderFilesSql } from '../resources'
+import {
+  Query,
+  ColumnMultiValueFunctionQueryFilter,
+  ColumnMultiValueFunction,
+  COLUMN_MULTI_VALUE_FUNCTION_QUERY_FILTER_CONCRETE_TYPE_VALUE,
+} from '@sage-bionetworks/synapse-types'
+import { cohortBuilderSql } from '../resources'
 
 export const handleUpsetPlotClick = (selection: any) => {
   //  We should use generateEncodedQueryForSelectedFacetURL, but this single facet may have multiple values (union from UpsetPlot)
   const sets = selection.sets
-  const facetColumnRequest: FacetColumnRequest = {
```

Comment:
Instead of creating a selectedFacets filter, use the additionalFilters.  Create n filters which will be AND'd together to find the intersection